api/status.go

```diff
@@ -70,6 +72,56 @@ func GetAllStatuses() (map[string]interface{}, *model.AppError) {
 	}
 }
 
+func getStatusesByIdsWebSocket(req *model.WebSocketRequest) (map[string]interface{}, *model.AppError) {
```

Comment:
We should move this to a normal web service call.  The web-socket is already overloaded in our latest load tests.